facts/4.2/archlinux-x86_64.facts

```diff
@@ -204,17 +427,6 @@
   "operatingsystem": "Archlinux",
   "os": {
     "architecture": "x86_64",
-    "distro": {
```

Comment:
hm, the whole os.distro path is missing, no idea if that is ok

Reply:
yes. facter 4 dropped that for arch linux. It was only there previously because the factset was copied from a facter 3 version and not properly generated.